src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -61,9 +61,10 @@ std::string ConvertPacketDataToString(const uint8_t* data,
 
 const size_t kStackSize = 131072;
 
-utils::SemanticVersion default_protocol_version(5, 2, 0);
+utils::SemanticVersion default_protocol_version(5, 3, 0);
 utils::SemanticVersion min_multiple_transports_version(5, 1, 0);
 utils::SemanticVersion min_cloud_app_version(5, 2, 0);
+utils::SemanticVersion min_reason_param_version(5, 3, 0);
```

Comment:
I've changed these version variables to `const` in https://github.com/smartdevicelink/sdl_core/pull/3462/commits/c85e87f6e19357f22feba51f0732a4238812ff60 but I'm not sure if `constexpr` would work here(since `utils::SemanticVersion` doesn't seem to be a literal type).